test/cases/ostree.sh

```diff
@@ -26,14 +26,12 @@ case "${ID}-${VERSION_ID}" in
         IMAGE_TYPE=rhel-edge-commit
         OSTREE_REF="rhel/8/${ARCH}/edge"
         OS_VARIANT="rhel8.3"
-        # When 8.3 was released, it wasn't available on all RH internal
-        # mirrors, therefore the Boston mirror is hardcoded.
-        BOOT_LOCATION="http://download.eng.bos.redhat.com/released/rhel-8/RHEL-8/8.3.0/BaseOS/x86_64/os/";;
+        BOOT_LOCATION="http://download.devel.redhat.com/released/rhel-8/RHEL-8/8.3.0/BaseOS/x86_64/os/";;
     "rhel-8.4")
         IMAGE_TYPE=rhel-edge-commit
         OSTREE_REF="rhel/8/${ARCH}/edge"
         OS_VARIANT="rhel8-unknown"
-        BOOT_LOCATION="http://download.devel.redhat.com/nightly/rhel-8/RHEL-8/latest-RHEL-8.4/compose/BaseOS/x86_64/os/";;
+        BOOT_LOCATION="$COMPOSE_URL/compose/BaseOS/x86_64/os/";;
```

Comment:
I think I'm confused by this PR - `COMPOSE_URL` isn't defined in `ostree.sh`, how is this going to work on a PR pipeline?

Reply:
Indeed you are right. This is at the top of prepare-rhel-internal.sh -> "${COMPOSE_URL:-http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/$COMPOSE_ID}"

I think we need to move this either as a separate script or define it in Jenkinsfile.  Or default to nightly here if COMPOSE_URL isn't set.

Reply:
Jenkinsfile would work, but don't know what value to put there as default... I guess separate script would work better as it could be called whenever and set the variable?